diff --git a/src/alloc.h b/src/alloc.h
index 04fb7e101..76f1867b7 100644
--- a/src/alloc.h
+++ b/src/alloc.h
@@ -12,14 +12,14 @@
 
 extern git_allocator git__allocator;
 
-#define git__malloc(len)                      git__allocator.gmalloc(len, __FILE__, __LINE__)
-#define git__calloc(nelem, elsize)            git__allocator.gcalloc(nelem, elsize, __FILE__, __LINE__)
-#define git__strdup(str)                      git__allocator.gstrdup(str, __FILE__, __LINE__)
-#define git__strndup(str, n)                  git__allocator.gstrndup(str, n, __FILE__, __LINE__)
-#define git__substrdup(str, n)                git__allocator.gsubstrdup(str, n, __FILE__, __LINE__)
-#define git__realloc(ptr, size)               git__allocator.grealloc(ptr, size, __FILE__, __LINE__)
-#define git__reallocarray(ptr, nelem, elsize) git__allocator.greallocarray(ptr, nelem, elsize, __FILE__, __LINE__)
-#define git__mallocarray(nelem, elsize)       git__allocator.gmallocarray(nelem, elsize, __FILE__, __LINE__)
+#define git__malloc(len)                      git__allocator.gmalloc(len, "libgit2", "0")
+#define git__calloc(nelem, elsize)            git__allocator.gcalloc(nelem, elsize, "libgit2", "0")
+#define git__strdup(str)                      git__allocator.gstrdup(str, "libgit2", "0")
+#define git__strndup(str, n)                  git__allocator.gstrndup(str, n, "libgit2", "0")
+#define git__substrdup(str, n)                git__allocator.gsubstrdup(str, n, "libgit2", "0")
+#define git__realloc(ptr, size)               git__allocator.grealloc(ptr, size, "libgit2", "0")
+#define git__reallocarray(ptr, nelem, elsize) git__allocator.greallocarray(ptr, nelem, elsize, "libgit2", "0")
+#define git__mallocarray(nelem, elsize)       git__allocator.gmallocarray(nelem, elsize, "libgit2", "0")
 #define git__free                             git__allocator.gfree
 
 /**
